kolibri/plugins/style_guide/vue-doc-loader.js

```diff
@@ -0,0 +1,22 @@
+const vuedoc = require('@vuedoc/parser');
+const path = require('path');
+const mkdirp = require('mkdirp');
+
+module.exports = function(content) {
```

Comment:
A brief top-level description about what this loader is doing, and how it should be used?